Orion.Net/Models/UserProfileViewModel.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Orion.Net.Models
+{
+    /// <summary>
+    /// Profile View Model of the AAD User
+    /// </summary>
+    public class UserProfileViewModel
```

Comment:
Please rename file and class to UserProfileModel.
ViewModel is prefered for Mvvm binding